server/protocols/jmap-rfc-8621/doc/specs/spec/quotas/quota.mdown

```diff
@@ -0,0 +1,157 @@
+# Quota
+
+The quota is an object that displays the limit set to an account usage as well as the current usage in regard to that limit.
+
+The quota object MUST contain the following fields:
+
+* **id**: `Id` The unique identifier for this object. It should respect the JMAP ID datatype defined in section 1.2 of [@!RFC8620]
+
+* **resourceType**: `ResourceType` The resource type of the quota.
+
+* **used**: `UnsignedInt` The current usage of the defined quota. Computation of this value is handled by the server.
+
+* **limit**: `UnsignedInt` The hard limit set by this quota. Objects in scope may not be created or updated if we reach this limit. It should be higher than the `warnLimit` and the `softLimit`.
+
+* **scope**: `Scope` The `Scope` of this quota.
+
+* **name**: `String` The name of the quota object. Useful for managing quotas and use queries for searching.
+
+* **datatypes**: `String[]` A list of all the data types values that are applying to this quota. This allows to assign
+quotas to separated or shared data types. This MAY include data types the client does not recognise. Clients MUST
+ignore any unknown data type in the list.
+
+The quota object MAY contain the following field:
+
+* **warnLimit**: `UnsignedInt|null` The warn limit set by this quota object. It can be used to send a warning to an
```

Comment:
Should we remark that `warnLimit = 0.9 * limit`?

Reply:
No, I think that is rather an implementation detail of a server than a spec detail

Reply:
No, I think it is more like an implementation detail of server rather than spec